themes/default/files/scripts/display.js

```diff
@@ -2316,6 +2316,8 @@ function showScoreFeedback(obj,settings)
 			}
 		}),
 		message: ko.computed(function() {
+			if(part.settings.showFeedbackIcon==false):

```

Comment:
Also, `part` is not defined in this scope. I think you just want `settings.showFeedbackIcon`.